tests/utilities_test.py

```diff
@@ -4,6 +4,9 @@
 import unittest
 
 import switch_mod.utilities as utilities
+import os
```

Comment:
One other nit:  It's good style to keep imports of standard libraries together in one block, so you could group this with "import unittest".